.rubocop.yml

```diff
@@ -199,6 +199,15 @@ Style/StringLiterals:
 Lint/RequireParentheses:
   Enabled: true
 
+Rails/CreateTableWithTimestamps:
+  Enabled: false
+
+Rails/HasManyOrHasOneDependent:
+  Enabled: false
+
+Rails/InverseOf:
+  Enabled: false
```

Comment:
@marderer why did you disable all those metrics?

Reply:
Having timestamps everywhere is a good practice 😉

Reply:
@AleksSenkou In this article https://launchschool.com/blog/understanding-polymorphic-associations-in-rails, wrote that `InverseOf` does not work with polymorfic, but rubocop required add it. 
About timestamps I asked in slack.

Reply:
@AleksSenkou I added timestamps and deleted cop.

Reply:
@marderer thank you, got it